src/app/Dashboard/AutomatedAnalysis/AutomatedAnalysisCard.tsx

```diff
@@ -740,24 +746,15 @@ export const AutomatedAnalysisCard: React.FunctionComponent<AutomatedAnalysisCar
       return undefined;
     }
     return (
-      <div
-        style={{
-          color: 'var(--pf-global--Color--200)',
-          fontWeight: 'lighter',
-          fontSize: '0.90em',
-          fontFamily: 'inherit',
-        }}
-      >
-        {`${usingArchivedReport ? 'Archived' : usingCachedReport ? 'Cached' : 'Active'} report name:`}{' '}
-        <span style={{ color: 'var(--pf-global--Color--100)', fontWeight: 'bold', fontStyle: 'italic' }}>{report}</span>
-      </div>
+      <Label icon={<InfoCircleIcon />} color={'cyan'}>
+        {`${usingArchivedReport ? 'Archived' : usingCachedReport ? 'Cached' : 'Active'} report name=${report}`}
+      </Label>
     );
   }, [usingArchivedReport, usingCachedReport, report, isLoading, errorMessage]);
 
   return (
-    <Card id="automated-analysis-card" isRounded isCompact isExpanded={isCardExpanded}>
+    <Card id="automated-analysis-card" isRounded {...props} isExpanded={isCardExpanded}>
```

Comment:
On that note, these ids would have to be generated to be unique somehow to both adhere to HTML and to help with having per-card filters.